### ISSUE TITLE
Finetuning on different dataset with same classes

### ISSUE THREAD

**Issue by ds2268** [facebookresearch/maskrcnn-benchmark#1052]
## ❓ Questions and Help
I have trained the model on dataset X with 1 class and want to finetune the model on dataset Y which has the same class. What is the most convenient way to do it as I don't need to do model surgery? I have final_model.pth and just want to finetune on it.

**Comment by Dorozhko-Anton** [comment]
How many classes do you have in total?

If only one I think you can do the following: 
1. initialize your model with your weights. To do so use the `MODEL.WEIGHT` parameter
```
MODEL:
  ...
  WEIGHT: "/path/to/final_model.pth"
  ...
```
2. Change LR and the number of iterations to smaller values than initial training

Some ideas of finetuning configs can be found in `quick` configs like this one https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/configs/quick_schedules/rpn_R_50_FPN_quick.yaml

3. [Optionally] freeze some layers see #1001

**Comment by ds2268** [comment]
@Dorozhko-Anton 

I have used the same config file where I have modified MODEL weight with an absolute path to the final_model.pth for finetuning. I have also changed output directory in this config file.

The result is the training instantly finishes (without any iterations)

019-08-26 11:01:21,021 maskrcnn_benchmark.utils.miscellaneous WARNING: Dataset [ShipDataset] has no categories attribute, labels.json file won't be created
2019-08-26 11:01:21,023 maskrcnn_benchmark.trainer INFO: Start training
2019-08-26 11:01:21,110 maskrcnn_benchmark.trainer INFO: Total training time: 0:00:00.085638 (0.0000 s / it)

Any guidance? Do I need to modify somehow the final_model.pth before finetuning?

Thanks in advance!

**Comment by Darshan2701** [comment]
@ds2268 You indeed have to do some model surgery and remove the "optimizer", "learning_rate" and "iterator" and then have to save the new model using torch.save(). Then you need to use this model for fine-tuning with different learning rate and related hyper params.
Please let me know if you have figured out, else I can post the code here.

**Comment by Darshan2701** [comment]
@ds2268 #652 Solves this.
